corehq/apps/cloudcare/views.py

```diff
@@ -441,6 +447,39 @@ def custom_json_handler(obj):
         },
         default=custom_json_handler,
     )
+@cloudcare_api
+def render_form(request, domain, session_id):
+    # get session
+    try:
+        session = EntrySession.objects.get(session_id=session_id)
+    except EntrySession.DoesNotExist:
+        session = None
+
+    response = requests.post(get_url_base() + '/webforms/get-xml/' + session_id)
+    json_response = json.loads(response.text)
```

Comment:
may want to ensure that the request received a 200 before parsing else this will fail hard